crates/aranya-daemon-api/src/service.rs

```diff
@@ -144,7 +160,10 @@ pub trait DaemonApi {
     async fn get_device_id() -> Result<DeviceId>;
 
     /// Adds the peer for automatic periodic syncing.
-    async fn add_sync_peer(addr: Addr, team: TeamId, interval: Duration) -> Result<()>;
+    async fn add_sync_peer(addr: Addr, team: TeamId, config: SyncPeerConfig) -> Result<()>;
+
+    /// Sync with peer immediately.
+    async fn sync_now(addr: Addr, team: TeamId, maybe_cfg: Option<SyncPeerConfig>) -> Result<()>;
```

Comment:
`Option<SyncPeerConfig>` is enough to know the parameter is optional without including `maybe` in the parameter name.
```suggestion
    async fn sync_now(addr: Addr, team: TeamId, cfg: Option<SyncPeerConfig>) -> Result<()>;
```